packages/server/src/trpc-routers/__tests_e2e__/swap-router.test.ts

```diff
@@ -0,0 +1,223 @@
+import { CoinPretty, Int, PricePretty, RatePretty } from "@keplr-wallet/unit";
+import { isNumeric } from "@osmosis-labs/utils";
+import { initTRPC } from "@trpc/server";
+
+import { AssetLists } from "../../queries/__tests__/mock-asset-lists";
+import { MockChains } from "../../queries/__tests__/mock-chains";
+import { createInnerTRPCContext } from "../../trpc";
+import { superjson } from "../../utils/superjson";
+import { swapRouter } from "../swap-router";
+
+const { createCallerFactory, router: createTRPCRouter } = initTRPC
+  .context<typeof createInnerTRPCContext>()
+  .create({
+    transformer: superjson,
+  });
+const createCaller = createCallerFactory(createTRPCRouter({ swapRouter }));
+const caller = createCaller({
+  assetLists: AssetLists,
```

Comment:
Prob worth adding a comment explaining that we're smoke testing against prod data from SQS and the mock asset/chain lists are hardcoded snapshots of the prod generated lists. So, perhaps we can point out that if you're troubleshooting an issue, perhaps these snapshots need to be updated.

Reply:
Thank you! This is a great suggestion. Also, because we are working on end-to-end tests, I'm considering fetching the asset and chain lists from our assetlist repo. I believe this approach will make these tests more resilient and help us avoid situations like this